libs/blocks/quiz-results/quiz-results.js

```diff
@@ -82,14 +83,10 @@ export default async function init(el, debug = null, localStoreKey = null) {
 
     loadFragments(el, basic);
   } else {
+    window.lana.log(`${loadingErrorText} The quiz-results block is misconfigured`);
```

Comment:
Nit:  Can we do optional chaining here `window.lana?.log`?

Reply:
@sabyamon a quick look through the rest of Milo shows a mix of both approaches - not sure what the recommended approach is, but most instances are just window.lana.log()